packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -515,25 +515,38 @@ export class PhoenixObjects {
   }
 
   /**
-   * Get the cuboid geometry for a Calo hit
-   * @param length length of the cuboid
-   * @param width width of the cuboid
-   * @param clusterParams Parameters for the Cluster (which must include color
+   * Get the cuboid geometry for a Calorimeter hit
+   * @param clusterParams  Parameters for the Cluster (can include color)
+   * @param defaultCellWidth width of the cuboid
+   * @param defaultCellLength length of the cuboid
    * @returns Geometry.
    */
   private static getCaloCube(
-    length: number,
-    width: number = 30,
-    clusterParams: any
+    clusterParams: any,
+    defaultCellWidth: number = 30,
+    defaultCellLength: number = 30
   ) {
-    if (length < width) {
-      length = width;
+    let cellWidth = clusterParams.side;
```

Comment:
You can use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) for both `cellWidth` and `cellLength`..

```ts
const cellWidth = clusterParams.side ?? defaultCellWidth;
```

Reply:
Thanks for the tip